internal/api/v1/user/configs/configs_test.go

```diff
@@ -0,0 +1,90 @@
+package configs_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	client "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigsAPI_SharedConfig(t *testing.T) {
+	httpmock.Activate()
+	t.Cleanup(httpmock.DeactivateAndReset)
+
+	tests := map[string]struct {
+		expectedErr error
+		expectedCfg *response.SharedConfig
+	}{
+		"shared config: HTTP GET /api/v1/configs/shared response: 200": {
+			expectedCfg: &response.SharedConfig{
+				PaymailDomains: []string{
+					"john.doe.test.4chain.space",
+				},
+				ExperimentalFeatures: map[string]bool{
+					"pikeContactsEnabled": true,
+					"pikePaymentEnabled":  false,
+				},
+			},
+		},
+		"shared config: HTTP GET /api/v1/configs/shared response: 500": {
+			expectedErr: models.SPVError{
+				Message:    http.StatusText(http.StatusInternalServerError),
+				StatusCode: http.StatusInternalServerError,
+				Code:       models.UnknownErrorCode,
+			},
+		},
+	}
+
+	transport := httpmock.NewMockTransport()
+	cfg := client.Config{
+		Addr:      "http://localhost:3003",
```

Comment:
This address could be placed in some fixtures package to be reused across the tests